### ISSUE TITLE
How to increase image tracing performance for round objects

### ISSUE THREAD

**Issue by egemenozkan** [jankovicsandras/imagetracerjava#6]
Hello,
0 - "Zero", O - "o", Circles and  alphabetic characters and numbers which includes rounded part, for example; "e", "8" which are not being image traced efficiently. Do you have suggestion for increasing image tracing performance for round objects ?

**Comment by jankovicsandras** [comment]
Hi,

TLDR; ☺

 - set ```ltres = 0.01``` and ```qtres = 1``` for round shapes
 - set ```ltres = 1``` and ```qtres = 0.01``` for straight shapes

e.g. from CLI:
```bash
java -jar ImageTracer.jar smiley.png outfilename smiley_output.svg ltres 0.01 qtres 1
```
or in Java:
```java
// Options
HashMap<String,Float> options = new HashMap<String,Float>();

// Tracing
options.put("ltres",0.01f);
options.put("qtres",1f);

ImageTracer.saveString(
	"output.svg" ,
	ImageTracer.imageToSVG( "input.jpg", options, null )
);
```
-----------------
The long story:

```ltres``` means **l**inear error **tres**hold, or how precise the coordinates should be for straight line segments. ```qtres``` means **q**uadratic error **tres**hold, the same for [quadratic splines](https://en.wikipedia.org/wiki/Quadratic_spline) (round arcs). 

If you set ```ltres``` to low values, e.g. 0.1 or 0.01 (and leave ```qtres``` the default 1), then the algorithm will go over the linear error treshold more often, and be forced to use more round shapes. Conversely, low ```qtres``` and high ```ltres``` will force more straight shapes. These values are pixel distance, so the reasonable value range is [ 0.0 ... 2.0 ] . I recommend experimenting with different ```ltres``` and ```qtres``` values.
More info: [Process overview](https://github.com/jankovicsandras/imagetracerjava/blob/master/process_overview.md)
